.env.example

```
@@ -221,6 +221,14 @@ EMAIL_USERNAME=your_email@example.com
 EMAIL_PASSWORD=your_app_password
 IMAP_SERVER=imap.migadu.com
 IMAP_TIMEOUT_SECONDS=10.0
+# Onboarding email sending for reviewed /onboarding-email send=true drafts
+ONBOARDING_EMAIL_SMTP_SERVER=smtp.migadu.com
+ONBOARDING_EMAIL_SMTP_PORT=465
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ONBOARDING_EMAIL_SMTP_PORT key should go before the ONBOARDING_EMAIL_SMTP_SERVER key

(UnorderedKey)